### PR TITLE
Finalize voxpupuli transition

### PR DESCRIPTION
**Pull request by h-haaks**
#### Pull Request (PR) description
Fix remaining metadata and readme to finalize the elastic -> voxpupuli transistion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+[![Build Status](https://github.com/voxpupuli/puppet-logstash/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-logstash/actions?query=workflow%3ACI)
+[![Release](https://github.com/voxpupuli/puppet-logstash/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-logstash/actions/workflows/release.yml)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/logstash.svg)](https://forge.puppetlabs.com/puppet/logstash)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/logstash.svg)](https://forge.puppetlabs.com/puppet/logstash)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/logstash.svg)](https://forge.puppetlabs.com/puppet/logstash)
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/logstash.svg)](https://forge.puppetlabs.com/puppet/logstash)
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-logstash)
+[![Apache-2 License](https://img.shields.io/github/license/voxpupuli/puppet-logstash.svg)](LICENSE)
+[![Donated by Elastic](https://img.shields.io/badge/donated%20by-Elastic-fb7047.svg)](#transfer-notice)
```
Comment (h-haaks) [inline README.md:11]: This was copied from the elastic_stack module.

**Comment by h-haaks** [comment]
@kenyon, @ekohl @bastelfreak Could you please have a look at this?
Is there anything more that has to be done here before we can release the first voxpupuli version?

**Review by kenyon — APPROVED** [review]

**Review by smortex — APPROVED** [review]

**Comment by h-haaks** [comment]
Could any of you merge this? I don't have merge privileges ..
I'll create a release branch as soon as this is merged.

**Comment by h-haaks** [comment]
Would be nice if someone could close #400 as well as it is resolved by #418

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Thanks @h-haaks, not just for this PR. Are you interested in joining Vox Pupuli so you can maintain the module?

**Comment by h-haaks** [comment]
Thanks @ekohl that would be cool :)

**Comment by bastelfreak** [comment]
@h-haaks I sent out the GitHub invitation :) Welcome onboard!
I recommend joining our IRC channel #voxpupuli on Libera or our Slack channel #voxpupuli on https://slack.puppet.com/

**Pull request merged**